prince-of-versions/quality.gradle

```diff
@@ -17,29 +16,6 @@ task checkstyle(type: Checkstyle) {
     classpath = files()
 }
 
-task findbugs(type: FindBugs) {
```

Comment:
Why did you remove findbugs? If it cannot be used anymore we can replace it with Spotbugs, which should be effectively the same.

Reply:
👌, will do